src/app/(customerFacing)/products/page.tsx

```diff
@@ -40,21 +56,27 @@ export default function ProductsPage({
               </>
             }
           >
-            <ProductsSuspense />
+            <ProductsSuspense options={options} />
           </Suspense>
```

Comment:
This one is from my laziness but we can also reduce number of lines by creating a component that maps through each skeleton.

```suggestion
<Suspense fallback={<LoadingSkeletons count={6} />}>
            <ProductsList options={sortOptions} />
          </Suspense>
```

Then create elsewhere

```suggestion
// Component to display loading skeletons while data is loading
function LoadingSkeletons({ count }: { count: number }) {
  return (
    <>
      {Array.from({ length: count }, (_, i) => (
        <ProductCardSkeleton key={i} />
      ))}
    </>
  );
}
```